src/main/java/lrgs/lrgsmain/LrgsMain.java

```diff
@@ -171,14 +173,22 @@ public void run()
         // Establish a server lock file & start the server lock monitor
 
         String lockName = EnvExpander.expand(cmdLineArgs.getLockFile());
-        myServerLock = new ServerLock(lockName);
-        if (!myServerLock.obtainLock(this))
+        Logger.instance().info("Lock File =" + lockName);
+        if (lockName.equals("-"))
```

Comment:
consider documenting the special hyphen argument in the argument usage.

Reply:
I have, though in the dev docs for now.